amethyst_renderer/src/renderer.rs

```diff
@@ -115,6 +119,16 @@ impl Renderer {
 
         self.device.cleanup();
 
+        // # Enforce maximum FPS
+        // Sleep if the time needed is > 10 ms for 9.5 ms
+        let time_remaining = Duration::from_secs(1) / self.max_fps - start.elapsed();
+        if time_remaining > Duration::new(0, 10000000) {
```

Comment:
I fear sleeping precision is worse than that.
In fact it may be so bad we shouldn't use it here at all.
Or do it for at least 100ms period.

Reply:
Well. Since you've tested it you may have more actual info about it than my guesding